libfio.c

```diff
@@ -101,7 +101,6 @@ static void reset_io_counters(struct thread_data *td, int all)
 
 	td->zone_bytes = 0;
 
-	td->last_was_sync = false;
```

Comment:
I think we never should have set `last_was_sync` to false here because when ramp time is over we want to continue to sync at the appropriate point, so there is no need to modify the value for `last_ddir_issued`. Agree?

Reply:
If you meant we don't need to modify the value of `last_ddir_issued` in this place, yes, I agree.